tests/unit/test_business_logic.py

```diff
@@ -2190,7 +2200,30 @@ def test_approve_file(bll):
         release_request,
         AuditEventType.REQUEST_FILE_APPROVE,
         user=checker,
-        path=path,
+        path=request_file.relpath,
+        group="group",
+    )
+
+
+def test_approve_file_group(bll):
```

Comment:
`test_approve_file_group` (and `test_reject_file_group` below) sound like they're testing approving a filegroup, rather than testing that the group appears in the audit when a file is approved. Maybe something like `test_approve_file_audit_event`?

Reply:
interesting - I've since modified the basic tests to verify the default group appears in the audit, this was testing that a manually specified group also appears in the audit log, which maybe made some sense when we were passing a group_name to the accept/reject methods - but now we're passing a RequestFile I actually don't think it adds anything. So I'm going to remove it entirely.